jmxtrans-core/src/main/java/com/googlecode/jmxtrans/JmxTransformer.java

```diff
@@ -492,6 +505,22 @@ private void deleteAllJobs() throws Exception {
 		}
 	}
 
+	private void uninitializeExecutors() throws MalformedObjectNameException {
+		this.uninitializeExecutors(queryExecutorRepository);
+		this.uninitializeExecutors(resultExecutorRepository);
+	}
+
+	private void uninitializeExecutors(ExecutorRepository executorRepository) {
+		executorRepository.clear();
+	}
+
+	private void restartSystem() throws Exception {
+		this.deleteAllJobs();
```

Comment:
I'm surprised by all those explicit `this.`. It looks somewhat coherent with the rest of the code in this class, but this is (I hope) not how all the code in this project looks. (Very minor comment, feel free to ignore it).

Reply:
Ha, I usually don't `this` when it's not needed, but I saw a lot of it in this file so I followed suit :) I'm adding a commit that removes *all* unnecessary `this` from this file.